esi/src/config.rs

```diff
@@ -5,16 +5,20 @@
 /// let config = esi::Configuration::default()
 ///     .with_namespace("app");
 /// ```
+#[allow(clippy::return_self_not_must_use)]
 #[derive(Clone, Debug)]
 pub struct Configuration {
     /// The XML namespace to use when scanning for ESI tags. Defaults to `esi`.
     pub namespace: String,
+    /// Disables URL unescaping, only good with non-HTML content, eg JSON
+    pub is_escaped: bool,
 }
```

Comment:
The purpose of this variable isn't immediately obvious to me from the name. Perhaps something like `content_is_escaped` or even more descriptive of its behaviour such as `should_escape_urls` would be more suitable?

Reply:
`is_escaped_content`? I think there is a convention to start `bool` variables with `is_` prefix...